From 31b371a67de2cd8e434c106fb25173e0590b76ef Mon Sep 17 00:00:00 2001
From: Muthukumar K <muthukumar.kandasamy@intel.com>
Date: Thu, 16 Aug 2012 10:03:24 +0530
Subject: [PORT FROM MAIN]Jellybean: WiDi to play at user selected volume level (Incorporate ICS Audio policy)

BZ: 52439

The volume level should be able to be adjusted from the device
itself for WiDi. So removing the limitation of always max
volume from policy manager.

Change-Id: I4f20d726ed2dd92ccef8347856b1009fc7b2044d
Orig-Change-Id: I913d54a47d37908ef36a794ef55a4f9d32614d44
Signed-off-by: Muthukumar K <muthukumar.kandasamy@intel.com>
---
 audio/AudioPolicyManagerBase.cpp |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/audio/AudioPolicyManagerBase.cpp b/audio/AudioPolicyManagerBase.cpp
index 91131cd..300ae58 100644
--- a/audio/AudioPolicyManagerBase.cpp
+++ b/audio/AudioPolicyManagerBase.cpp
@@ -2861,7 +2861,6 @@ float AudioPolicyManagerBase::computeVolume(int stream,
     if (stream == AudioSystem::MUSIC &&
         index != mStreams[stream].mIndexMin &&
         (device == AUDIO_DEVICE_OUT_AUX_DIGITAL ||
-         device == AUDIO_DEVICE_OUT_WIDI ||
          device == AUDIO_DEVICE_OUT_DGTL_DOCK_HEADSET ||
          device == AUDIO_DEVICE_OUT_USB_ACCESSORY ||
          device == AUDIO_DEVICE_OUT_USB_DEVICE)) {
-- 
1.7.4.1

